server/src/main/java/io/unitycatalog/server/persist/TableRepository.java

```diff
@@ -204,7 +192,10 @@ public static String getNextPageToken(List<TableInfoDAO> tables) {
             return "";
         }
         // Assuming the last item in the list is the least recent based on the query
-        return String.valueOf(tables.get(tables.size() - 1).getUpdatedAt().getTime());
+        long time = tables.get(tables.size() - 1).getCreatedAt().getTime();
```

Comment:
why are we using update time as the page token? that seems unstable. Do we sort the returned table items using the last update time? If so, is there an index for that?

Reply:
Yes, there is [an index](https://github.com/unitycatalog/unitycatalog/blob/main/server/src/main/java/io/unitycatalog/server/persist/dao/TableInfoDAO.java#L15) on updated time.  And, the returned items are [sorted based on the update time.](https://github.com/unitycatalog/unitycatalog/blob/main/server/src/main/java/io/unitycatalog/server/persist/TableRepository.java#L230). cc: @ravivj-db

Reply:
Sure, we could do this, will add this as a item for a follow up PR?

Reply:
sounds good. looks like we're updating the pagination logic in the force delete PR too: https://github.com/unitycatalog/unitycatalog/pull/61/files.  

@ravivj-db can we separate out the pagination change ?